teps/0096-pipelines-v1.md

```diff
@@ -332,6 +332,15 @@ Each V1 CRD should be fully documented before being released, but release of a V
 unrelated documentation issues.
 Once we've achieved the goals of this TEP, we can release a v1 software version of Pipelines.
 
+### Migration plan
+We will start serving v1 versions after the initial release with [v1 CRDs](https://github.com/tektoncd/pipeline/pull/5579).
+And then one release later, we will swap the stored version to v1, mark v1beta1 as deprecated, and document that the EOL
+for v1beta1 is 9 months after that release.
```

Comment:
Do we want to remove support for `v1beta1` only 9 month after the release ? Especially since we did support `v1alpha1` for.. about a year I think.

Reply:
9 months doesn't sound too aggressive to me. We will provide 1 year of support for the last LTS release prior to the v1beta1 removal, so I'd think that should be plenty of time for users that want to stick to v1beta1 for sometime longer.

Reply:
Sounds fair 🙃